MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -124,7 +124,7 @@ docker build --no-cache -t opea/embedding-multimodal:latest --build-arg https_pr
 ### 2. Build retriever-multimodal-redis Image
 
 ```bash
-docker build --no-cache -t opea/retriever-multimodal-redis:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/retrievers/multimodal/redis/langchain/Dockerfile .
+docker build --no-cache -t opea/retriever-multimodal-redis:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/retrievers/redis/langchain/Dockerfile .
```

Comment:
Do we still need this `retriever-multimodal-redis` image?

Reply:
This file also need to update https://github.com/opea-project/GenAIExamples/blob/da8c92e7c5c855b660a44569012dee71b513770c/MultimodalQnA/docker_image_build/build.yaml#L32

Reply:
Also check this one, https://github.com/opea-project/GenAIExamples/blob/da8c92e7c5c855b660a44569012dee71b513770c/MultimodalQnA/docker_compose/intel/hpu/gaudi/compose.yaml#L77

Reply:
The image name should be unified to `retriever-redis`. Will fix this.